Classes/Settings/SettingsViewController.swift

```diff
@@ -162,19 +162,17 @@ NewIssueTableViewControllerDelegate, DefaultReactionDelegate {
     }
 
     func onReportBug() {
-        guard let viewController = NewIssueTableViewController.create(
-                client: GithubClient(userSession: sessionManager.focusedUserSession),
-                owner: "GitHawkApp",
-                repo: "GitHawk",
-                signature: .bugReport
-            ) else {
-                Squawk.showGenericError()
-                return
-        }
-        viewController.delegate = self
-        let navController = UINavigationController(rootViewController: viewController)
-        navController.modalPresentationStyle = .formSheet
-        present(navController, animated: trueUnlessReduceMotionEnabled)
+        showTemplateOptions()
+    }
+
+    func showTemplateOptions() {
+        client.createNewIssue(
```

Comment:
To make sure I'm clued up on what's happening here, so tapping on "Report a Bug" is now going to ask you which template to use "Bug Report" or "Feature Request", is that correct?

I'm easy but is this the functionality we want? Maybe we should rename the button or maybe default to the Bug Report template, or if people are fine with this we can keep the same!

Reply:
I don't have a preference,  switching to Report Bug/ Feature request sounds good or defaulting to bug. Currently, we don't have a quick link for feature request in settings. Maybe now we just offer both, bug report + feature request.

Reply:
It's a weird one, as a user I want to be able to do both easily! I'm thinking a button rename would be the best, but I'm lost on the naming for it!

Reply:
"Create an issue"?